src/gui/editors/AutomationEditor.cpp

```diff
@@ -170,28 +168,28 @@ AutomationEditor::AutomationEditor() :
 						SLOT( verScrolled( int ) ) );
 
 	// init pixmaps
-	if( s_toolDraw == NULL )
+	if (s_toolDraw == nullptr)
 	{
 		s_toolDraw = new QPixmap( embed::getIconPixmap(
 							"edit_draw" ) );
 	}
-	if( s_toolErase == NULL )
+	if (s_toolErase == nullptr)
 	{
 		s_toolErase= new QPixmap( embed::getIconPixmap(
 							"edit_erase" ) );
 	}
-	if( s_toolSelect == NULL )
+	if (s_toolDrawOut == nullptr)
 	{
-		s_toolSelect = new QPixmap( embed::getIconPixmap(
-							"edit_select" ) );
+		s_toolDrawOut = new QPixmap(embed::getIconPixmap(
+							"edit_draw_outvalue"));
```

Comment:
This only ends at column 81 (on 4 space tabs), I don't think that's enough reason to keep the odd formatting this section has.
```suggestion
		s_toolDrawOut = new QPixmap(embed::getIconPixmap("edit_draw_outvalue"));
```